### DISCUSSION TITLE
[HELP] The result is distorded

### DISCUSSION THREAD

**Discussion by Nalofox** [arkypita/LaserGRBL#1737] [Q&A]
Hi Everybody,

I'm Nathan, an automatician and I'm here to get help on a problem with a custom laser engraver.

I  built a laser engraver to make customized 3 by 3 round stainless steel water bottles (with 3 lasers in line).
When I want to engrave a water bottle, it starts cleanly, but after a while the line shifts (with and without filling).
I have tried many options, but nothing works as requested.

Here is a test on PVC (it's the same result on a bottle).

- Wanted :

![execpted2](https://user-images.githubusercontent.com/73492926/152639777-09d709b2-6efc-41c3-83bb-fc89c40fa8a7.PNG)
![execpted](https://user-images.githubusercontent.com/73492926/152639785-de02aef1-77f1-4d44-bc22-cf75e29ffe14.PNG)

- Reality : 
  - Laser 3

![IMG_20220205_114214](https://user-images.githubusercontent.com/73492926/152639810-41f4a898-8eea-4f4f-a4b4-dfa007b0f84b.jpg)

  - Laser 2

![IMG_20220205_114210](https://user-images.githubusercontent.com/73492926/152639812-8a8cb9de-1497-4620-8498-cb09b8f6f313.jpg)

  - Laser 1

![IMG_20220205_114212](https://user-images.githubusercontent.com/73492926/152639813-49e94040-1531-4e00-989c-c10df41605de.jpg)

Note that all 3 lasers receives the same TTL signal and if we engrave the same design with same configuration, the result is different on each occurence.

Here is the engraver configuration :
- 3 lasers syncronised and piloted by an TTL signal
- 1 nema 23 for the rotation with belt drive
- 1 nema 23 for the lasers moves
- CNC controller is a makerbase mks dlc v2.0 with 2 external nema 23 driver (MKS TMC2160)

Picture : [https://imgur.com/a/XwitpV4](https://imgur.com/a/XwitpV4)
Config : [https://imgur.com/a/5fvQwcd](https://imgur.com/a/5fvQwcd)

My question is why at a given moment there is a lag...

Thanks for your help.

Nathan

**Comment by gmmanonymus111** [comment]
Hy,
Well, for me, it seems your rotary axis, more precisely, the material on your rotary axis slips due to high speed and/or high acceleration. Try slower, lower your acceleration to around 50, or try some rubberized rod as your rotary axis, like [this](https://ae01.alicdn.com/kf/HTB1VBuckDZmx1VjSZFGq6yx2XXa1/EleksMaker-EleksRotate-Rotate-Engraving-Module-A3-Laser-Engraver-Y-Axis-DIY-Update-Kit-with-stepp-for.jpg_Q90.jpg_.webp)

**Reply by Nalofox** [reply]
Dear gmmanonymus111,

I have tired with acceleration set at 50 but still shift.
I will try yo had heat-shrinkable sleeve.

Thank you for help
